front-end/studio/src/app/pages/apis/{apiId}/editor/_services/rest-resource.service.ts

```diff
@@ -71,6 +71,7 @@ export class RestResourceService {
         // The GET operation
         let baseGetOp: OasOperation = basePathItem.createOperation("get");
         basePathItem.get = baseGetOp;
+        baseGetOp.deprecated = false;
```

Comment:
Do you think we need to add these?  The default value for `deprecated` is `false` anyway.